RELEASES.rst

```diff
@@ -521,4 +521,4 @@ Version 0.1.6 (2015-06-04)
 Version 0.1.4 (2015-05-20)
 ==========================
 
-- Add back DfsIter, BfsIter
\ No newline at end of file
+- Add back DfsIter, BfsIter
```

Comment:
FWIW, this was the GitHub editor's doing -- I think it's a "no newline at end of file" change.